lib/commands/tunnel.ts

```diff
@@ -66,7 +74,7 @@ export default class TunnelCmd extends Command {
 		'$ balena tunnel myApp -p 22222:0.0.0.0:222',
 		'',
 		'# multiple port tunnels can be specified at any one time',
-		'$ balena tunnel myApp -p 8080:3000 -p 8081:9000',
+		'$ balena tunnel myApp -p 8080:3000 8081:9000',
```

Comment:
The second `-p` is optional. I've removed it because it kind of clarifies the reason why the `-p` option must be provided at the end of the command line. (Which is really an oclif design bug, if you ask me...)

Reply:
Ah.  I wonder if this spaced separated convention might be confusing for people who have seen the comma separated convention we have adopted for multi arguments?

Reply:
Most recently we used the space separated syntax for the `--release-tags` option in the `push` and `deploy` commands:
```
$ balena help push
EXAMPLES
...
  $ balena push myApp --release-tag key1 "" key2 "value2 with spaces"

$ balena help deploy
EXAMPLES
...
  $ balena deploy myApp myApp/myImage --release-tag key1 "" key2 "value2 with spaces"
```
If it's controversial, I will revert the example.

Reply:
I have already reverted the change made in this PR to the usage example of the `-p` option.